pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -306,6 +312,27 @@ func (ssc *defaultStatefulSetControl) updateStatefulSet(
 		return set.Status.DeepCopy(), err
 	}
 
+	var isPreDownloadDisabled bool
```

Comment:
`isPreDownloadDisabled` should be a global variable in `statefulset` package and determined during controller initializing.